### PR TITLE
refactor: update top-side-2.tsx

### LINKED ISSUE #41147: Warning: [antd: Breadcrumb] `Breadcrumb.Item and Breadcrumb.Separator` is deprecated. Please use `items` instead.
### Reproduction link

[https://4x.ant.design/components/breadcrumb/](https://4x.ant.design/components/breadcrumb/)

### Steps to reproduce

My component works very well but I'm having this warning in the console. I've tried to resolve it with the new syntax in the page but is not working. Seems is not correct the documentation or anything else.

### What is expected?

-

### What is actually happening?

My component works very well but I'm having this warning in the console. I've tried to resolve it with the new syntax in the page but is not working. Seems is not correct the documentation or anything else.

| Environment | Info |
| --- | --- |
| antd | 5.3.0 |
| React | 18.2.0 |
| System | Linux, Mac,Windows |
| Browser | Chrome |

---

```tsx
import React from 'react';
import { Breadcrumb, Col, Row } from 'antd';
import { NavLink, useLocation } from 'react-router-dom';

const capitalizeChar = (str) => {
  return str.charAt(0).toUpperCase()   str.slice(1);
};

/**
 * This component uses react-router-dom
 * (should be used inside a Provider)
 */
const Breadcrumbs = () => {
  const location = useLocation();
  const pathSnippets = location.pathname.split('/').filter((i) => i);
  const breadcrumbItems = pathSnippets.map((snippet, index) => {
    const url = `/${pathSnippets.slice(0, index   1).join('/')}`;
    const title = snippet.replace(/-/g, ' ');
    return (
      <Breadcrumb.Item key={url}>
        <NavLink to={url}>{capitalizeChar(title)}</NavLink>
      </Breadcrumb.Item>
    );
  });

  return (
    <>
      <Row>
        <Col className="flex items-center gap-2 p-4">
          <Breadcrumb>
            <Breadcrumb.Item>
              <NavLink to="/">Home</NavLink>
            </Breadcrumb.Item>
            {breadcrumbItems}
          </Breadcrumb>
        </Col>
      </Row>
    </>
  );
};

export { Breadcrumbs };
```

### PR DESCRIPTION
**Pull request by Aimony**
Resolve the warning of console output "Warning: [antd: Breadcrumb] `Breadcrumb.Item and Breadcrumb.Separator` is deprecated. Please use `items` instead."

### 🤔 This is a ...

- [x] 🛠 Refactoring

### 🔗 Related Issues

close #41147

### CODE REVIEW

**Review by afc163 — DISMISSED** [review]

File: components/layout/demo/top-side-2.tsx [hunk 1]
```diff
@@ -58,11 +58,14 @@ const App: React.FC = () => {
...
-          <Breadcrumb style={{ margin: '16px 0' }}>
-            <Breadcrumb.Item>Home</Breadcrumb.Item>
-            <Breadcrumb.Item>List</Breadcrumb.Item>
-            <Breadcrumb.Item>App</Breadcrumb.Item>
-          </Breadcrumb>
+          <Breadcrumb
+            items={[
+              { title: 'Home' },
+              { title: 'List' },
+              { title: 'APP' }
```
Comment (afc163) [inline components/layout/demo/top-side-2.tsx:65]:
```suggestion
              { title: 'APP' },
```

**Review by afc163 — DISMISSED** [review]

File: components/layout/demo/top-side-2.tsx [hunk 2]
```diff
@@ -58,11 +58,14 @@ const App: React.FC = () => {
...
-          <Breadcrumb style={{ margin: '16px 0' }}>
-            <Breadcrumb.Item>Home</Breadcrumb.Item>
-            <Breadcrumb.Item>List</Breadcrumb.Item>
-            <Breadcrumb.Item>App</Breadcrumb.Item>
-          </Breadcrumb>
+          <Breadcrumb
+            items={[
+              { title: 'Home' },
+              { title: 'List' },
+              { title: 'APP' },
```
Comment (li-jia-nan) [inline components/layout/demo/top-side-2.tsx:65]:
```suggestion
              { title: 'App' },
```

snap 挂了

**Review by li-jia-nan — DISMISSED** [review]

**Comment by li-jia-nan** [comment]
you need update snap

**Comment by li-jia-nan** [comment]
run command `npm run test:update components/layout` please.

**Comment by Aimony** [comment]
> run command `npm run test:update components/layout` please.

Hello, is it running locally and resubmitting? This is my first time submitting a PR.

**Comment by li-jia-nan** [comment]
> Hello, is it running locally and resubmitting? This is my first time submitting a PR.

Yes, you are right

**Comment by Aimony** [comment]
> > Hello, is it running locally and resubmitting? This is my first time submitting a PR.
> 
> Yes, you are right

I will submit a new PR again.😀

**Comment by li-jia-nan** [comment]
> I will submit a new PR again.😀

That is not necessary, you can just submit it again on this PR.

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**